docs/source/guides/integrations/basic_aim_tensorflow_event_conversion.md

```diff
@@ -0,0 +1,83 @@
+### Show TensorFlow events in Aim 
+
+Aim give your possibility to convert TensorFlow event files into native format 
+and show them directly inside the Aim interface.
+
+Before showing the events in aim, the event files have to pass 
+the conversion process.
+
+Please note that only the following TF plugins are currently supported 
+- scalar
+- image
+
+To convert TensorFlow events, `aim convert` commant must be run on your log directory.
+
+```commandline
+aim convert --repo .aim tf --logdir ~/tensorflow/logdir
```

Comment:
better to leave out `.aim` (it is handled automatically)
`aim convert --repo aim_repo tf --logdir ~/tensorflow/logdir` or
`aim convert --repo . tf --logdir ~/tensorflow/logdir`

Reply:
I removed `--repo` at all since it's handled automatically